src/open_inwoner/scss/components/Form/Form.scss

```diff
@@ -169,3 +169,11 @@
     }
   }
 }
+
+/// Registration.
+
+#registration-form .form__control label.label ul {
```

Comment:
IIRC there was some sort of rule/method to not use element names in the selectors, something with specificity (hence why everything has class names matching the tags).

Reply:
@Bartvaderkin I've changed it now - it's a unique situation that i think only occurs on the Registration page. + it's not valid HTML to have a UL tag (=block) inside a label (=inline) - but this is quick fix for now.